### PR TITLE
docs: AOT guide for CLI #19510

### PR DESCRIPTION
**Pull request by wardbell**
See issue #19510

This PR includes copy of aot-compiler.md from PR #19818 (SHA ca214db) because this PR deletes ToH pt-6 files referenced by the current aot-compiler.md.
These two  PRs must be coordinated.  If there is a conflict, **keep PR #19818** and discard this one.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by filipesilva — APPROVED** [review]

File: aio/content/guide/aot-compiler.md [hunk 1]
```diff
@@ -72,517 +77,898 @@ AOT compiles HTML templates and components into JavaScript files long before the
...
+
+<h3 class="no-toc">Symbol reference expected</h3>
+
+The compiler expected a reference to a symbol at the location specified in the error message.
+
+This error can occur if you use an expression in the `extends` clause of a class.
+
+<!--
+
+Chuck: After reviewing your PR comment I'm still at a loss. See [comment there](https://github.com/angular/angular/pull/17712#discussion_r132025495).
```
Comment (Foxandxss) [inline aio/content/guide/aot-compiler.md]: ????

File: aio/content/guide/aot-compiler.md [hunk 2]
```diff
@@ -72,517 +77,898 @@ AOT compiles HTML templates and components into JavaScript files long before the
...
+
+-->
+
+## Conclusion
 
-Here's the map for _Tour of Heroes_.
+This page covered:
```
Comment (Foxandxss) [inline aio/content/guide/aot-compiler.md]: We are doing `## Summary` now in ToH

**Review by Foxandxss — APPROVED** [review]

**Comment by Foxandxss** [comment]
The docs style guide was using some of this images so I put them again in the docs-style-guide folder.

**Comment by vicb** [comment]
missing:
- target label
- merge label
- green CI

**Review by mhevery — APPROVED** [review]

**Pull request merged**